### DISCUSSION TITLE
GLKit not working in mac catalyst

### DISCUSSION THREAD

**Discussion by Aniketbokre97** [HaishinKit/HaishinKit.swift#1187] [Q&A]
Im using hainshinkit framework for live streaming . I have used Replaykit to share screen on live streaming server.
In iOS and iPad app is working properly but when i run the app in mac Catalyst there is error in GLKit framework and app cant build. I have mac air m1.
![Screenshot 2023-04-19 at 10 31 21 AM](https://user-images.githubusercontent.com/105269069/232971442-7f971dfe-ea26-4eea-bfb9-5bc1f8e8200e.png)

Expectation: I want to work app like iOS and iPad in macOS device.

**Comment by shogo4405** [comment]
Please see this discussion. https://github.com/shogo4405/HaishinKit.swift/discussions/1010

**Comment by Aniketbokre97** [comment]
When i run app in iphone, MTHKView i.e Metalkit is used but when i run app in mac catalyst app crash in GHLKView i.e GLKit  . how to use MetalKit instead of GHLKit.

can you please give me the solution to use MTHKview in mac catalyst.
